week-1/explore/objects.md

```diff
@@ -5,12 +5,20 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+`console.log`
 
 Now enter just `console` in the Console, what output do you get back?
+`console` 
 
 Try also entering `typeof console`
 
 Answer the following questions:
 
 What does `console` store?
 What does the syntax `console.log` or `console.assert` mean? In particular, what does the `.` mean?
+
+The console.log() method prints a message to the web console. 
+
+The console.assert() method writes an error message to the console if the assertion is false. If the assertion is true, nothing happens.
+
+`.` means execute this”. 
```

Comment:
You can read this article for dot notation.
https://www.freecodecamp.org/news/dot-notation-vs-square-brackets-javascript/